code/datums/tutorial/_tutorial_menu.dm

```diff
@@ -79,5 +79,5 @@
 				return
 
 			path = new path
-			path.start_tutorial(usr)
-			return TRUE
```

Comment:
why remove this line?

Reply:
I figured that if the UI is going to close immediately after the tutorial starts, there wouldn't be much point in it requesting an update.

Reply:
I'd argue it's a good convention to keep

Reply:
I'll defer to you on this. 👍